addons/core/.template-lintrc.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 module.exports = {
-  extends: 'octane',
```

Comment:
Changed all of them from `octane` to `recommended` because it's just [an alias](https://github.com/ember-template-lint/ember-template-lint/blob/v3.12.0/lib/config/octane.js) for `recommended`. It's also removed in `4.0`.